Orange/widgets/data/oweditdomain.py

```diff
@@ -3080,4 +3132,5 @@ def column_str_repr_string(
 
 
 if __name__ == "__main__":  # pragma: no cover
-    WidgetPreview(OWEditDomain).run(Orange.data.Table("iris"))
+    WidgetPreview(OWEditDomain).run(Orange.data.Table(
+        "/Users/ajda/Desktop/test.csv"))
```

Comment:
Don't forget to remove this! I suggest you remove it now, and when you're making commits, add with `add -p` and skip this chunk.